tests/features_visualizations/test_regularizers.py

```diff
@@ -18,9 +42,9 @@ def test_lp_norm():
 
 
 def test_tv():
-    vec = tf.cast([[[0., 0., 0.],
-                    [0., 2., 0.],
-                    [0., 0., 1.]]], tf.float32)[:, :, :, tf.newaxis]
+    vec = tf.cast([[[0.0, 0.0, 0.0], [0.0, 2.0, 0.0], [0.0, 0.0, 1.0]]], tf.float32)[
```

Comment:
Even though it is better from a syntaxic point of view I think it is harder to know what this is doing at a glance with this format